docs/enhancement-proposals/off-ep-0005.md

```diff
@@ -0,0 +1,134 @@
+* The solvent dielectric constant was not specified.
+* Cutoffs were not specified
+* The physical constants used to compute the potential were unspecified
+
+To solve these issues, this OFF-EP proposes:
+* The `method` attribute is replaced with `periodic_potential` in analogy to other parameters that use the `potential` term to specify the functional form or a common choice
+* The `periodic_potential` attribute defaults to `Ewald3D-ConductingBoundary` as a valid keyword that states that the Ewald periodic sum with conducting boundary conditions should be the true potential used for periodic systems
+* PME (and other methods) are permissible approximations to Ewald as long as they are controlled.
+* For reaction field or other methods, the `periodic_potential` can specify the exact functional form used for the periodic potential or a keyword denoting a common choice, along with the optional `cutoff` and `solvent_dielectric` attributes
+* The `nonperiodic_potential` attribute defaults to `Coulomb` indicating the Coulomb potential is to be used in non-periodic systems, though other functional forms are accepted.
+* The `exception_potential` attribute defaults to `Coulomb`, indicating the Coulomb potential is to be used for exceptions, though other functional forms are accepted.
+* We explicitly specify which self-consistent physical constants should be used to compute
+
+We leave these to future extensions:
+* We reserve the possibility of adding a `nonperiodic_potential` keyword at a future date should it become necessary to permit different choices for non-periodic systems
+* We reserve the possibility of adding specific keywords to specify reaction field potentials for `periodic_potential`
+
+## Usage and Impact
+
+Since most force fields use some flavor of PME for periodic systems and something similar to
+`nonperiodic_potential="Coulomb"` for non-periodic systems, the default attributes for this tag will
+likely be the most commonly-used. Splitting `method` out into explicit attributes to specify periodic,
+nonperiodic, and exception potential energy terms, however, makes it less
+ambiguous how electrostatics should be handled in each case and decouples the method used in each
+case.
+
+Users are recommended to consider upgrading from the default attribute values of 0.3 to 0.4 to avoid
+continuing to use this ambiguity. Implementations may wish to execute this up-conversion automatically (see below).
+
+If backward-compatibility is provided as specified below, users of old force fields will not need to update their force field definitions.
+
+## Backward compatibility
+
+To minimize pain in implementations, we can allow implementations continue to accept `method="PME"` as a synonym for `periodic_potential="Ewald3D-ConductingBoundary"` for a few releases,
+issuing a warning that the spec has changed and this behavior may be removed in a future release.
```

Comment:
(not blocking) Thankfully, I don't think this is a concern. Since this change will be accompanied by an `Electrostatics` section version change, implementations will know which keywords are expected from each version. My plan for the Toolkit would be to just automatically up-convert any Electrostatics section that is read from 0.3 to 0.4 (though if there are strong objections I could try to implement round trip support for both versions, but that would be expensive, brittle, and complex).